src/main/java/org/alfresco/repo/domain/schema/SchemaBootstrap.java

```diff
@@ -951,6 +1007,11 @@ private boolean updateSchema(Connection connection) throws Exception
      */
     private void initialiseActivitiDBSchema(Connection connection)
     {
+        if (logger.isInfoEnabled())
+        {
+            logger.info("Initialise Activiti DB Schema tables");

```

Comment:
If this is before the fact, then it should be a statement of intention:
"Initialising Activiti DB schema tables."